packages/js/data/src/onboarding/types.ts

```diff
@@ -75,7 +75,7 @@ export type OnboardingState = {
 	profileItems: ProfileItems;
 	taskLists: Record< string, TaskListType >;
 	paymentMethods: Plugin[];
-	productTypes: OnboardingProductType[];
```

Comment:
Onboarding `productTypes` is an object of objects.

![Screenshot 2023-04-14 at 16 59 01](https://user-images.githubusercontent.com/4344253/231997021-eb7bc0e3-1493-43f1-a987-d890e23f8816.png)

![Screenshot 2023-04-14 at 16 11 30](https://user-images.githubusercontent.com/4344253/231996866-e7b1d838-01b7-44ff-8453-13a278279502.png)